WPThemeReview/Sniffs/Privacy/NoUrlShortenersSniff.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * WPThemeReview Coding Standard.
+ *
+ * @package WPTRT\WPThemeReview
+ * @link    https://github.com/WPTRT/WPThemeReview
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WPThemeReview\Sniffs\Privacy;
```

Comment:
Is this category correct ?

Reply:
Yes: https://make.wordpress.org/themes/handbook/review/required/#privacy

> No URL shorteners used in the theme.